include/circt/Dialect/FIRRTL/FIRRTLTypesImpl.td

```diff
@@ -578,4 +578,11 @@ def DoubleImpl : PropImplType<"Double"> {
   let typeName = "firrtl.propDouble";
 }
 
+def FStringTimeImpl : FIRRTLImplType<"FStringTime", [], "::circt::firrtl::FIRRTLType" > {
+  let summary = "A format string type that represents the current simulatoin time";
```

Comment:
```suggestion
  let summary = "A format string type that represents the current simulation time";
```

Reply:
Accidentally fixed via elision in: https://github.com/llvm/circt/pull/8342/commits/3abc9fdcf762c827344667c13a6c258d533db2ca